tests/behat/features/sitewidecontentreport.feature

```diff
@@ -4,13 +4,12 @@ Feature: Site wide content report
 
   Background:
 
-    Given the "group" "EDITOR group" has permissions "CMS_ACCESS_LeftAndMain"
-    And a "page" "My page" has the "Content" "<p>My content</p>"
+    Given a "page" "My page" has the "Content" "<p>My content</p>"
     And a "file" "test1.pdf"
-
-    Given I am logged in with "ADMIN" permissions
+    And the "group" "EDITOR" has permissions "Access to 'Pages' section" and "Access to 'Reports' section" and "Access to 'Subsites' section" and "Access to 'Files' section" and "FILE_EDIT_ALL"
```

Comment:
Are _all_ of these permissions required for every scenario? Seems likely there could be multiple groups here with for just the permissions each scenario explicitly requires.